### ISSUE TITLE
Attach databases limit, custom sqlite3 binary

### ISSUE THREAD

**Issue by Gofer33** [mapbox/node-sqlite3#1225]
TL;DR: How to set limit of attached databases to more than 10?

Sqlite3 by default has limits of attached databases per connection set to 10. I need to attach more than 10 and the easiest work around for me is to modify source code of sqlite and set limit to (for example) 100. 

I did this using this:
https://www.sqlite.org/amalgamation.html

and as a result i have my own modified sqlite3 binary. When i use sqlite3 in terminal i can attach more than 10 databases and it works fine.

But i could not manage to use this feature in node-sqlite3. 

I tried all sort of variations of commands like this:

npm install sqlite3 --build-from-source --sqlite=/usr/bin/sqlite3
(my modified binary is in /usr/bin/sqlite3)

and it always ends up installing default sqlite from node-pre-gyp and i still have attach limit set to 10;

I also tried to overwrite node_sqlite3.node by my binary but i got this error:

Error: /var/www/backend/node_modules/sqlite3/lib/binding/node-v57-linux-arm/node_sqlite3.node: cannot dynamically load executable
    at Object.Module._extensions..node (module.js:682:18)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/www/backend/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)

I am working on Rapsberry Pi and my OS is Raspbian GNU/Linux 9 (stretch)

**Comment by Gofer33** [comment]
nvm, i found work around for this
